_docs/releasing_version.md

```diff
@@ -49,6 +47,10 @@ the changelog generator expects the token in the environment variable `CHANGELOG
 CHANGELOG_GITHUB_TOKEN='mytoken' bundle exec rake changelog
 ```
 
+The changelog generator checks for certain labels on closed issues and PRs since the last release and groups them together. Backwards incompatible changes require a major release.
+If only bug fixes were merged, a patch release is suggested. If both isn't the case, continue with a minor release. Check the generated diff for the CHANGELOG.md. If your chosen
```

Comment:
Not quite following `If both isn't the case`. Both what?

Reply:
ah. What I meant: If the Changelog does not
* only contain merged PRs that are labeled `bug` 
* only contain merged PRs that are labeled `backwards-incompatible`

then go on with a minor release

Reply:
let me know if you've a better wording in mind :)

Reply:
> If the changes were neither backwards-incompatible nor only bug fixes, make a minor release.

Reply:
updated it